plugins/woocommerce/bin/package-update.sh

```diff
@@ -34,8 +34,6 @@ if [ -z "$SKIP_UPDATE_TEXTDOMAINS" ]; then
 	output 2 "Done!"
 
 	output 3 "Updating package JS textdomains..."
-	find ./packages/woocommerce-blocks \( -iname '*.js' -o -iname '*.json' \) -exec sed -i.bak -e "s/'woo-gutenberg-products-block'/'woocommerce'/g" -e "s/\"woo-gutenberg-products-block\"/\"woocommerce\"/g" {} \;
```

Comment:
I think it might be good to comment out at least one of these lines instead of removing it. That way this script still has the prior art that can be used to inform anything in the future (if there is anything) that might need this.

Otherwise, I think it makes more sense to just remove this script entirely. There's not really a need for it, right? Are there any strings in `action-scheduler` that need replaced? If so, is there an easier or more effective way we could do that which limits the need for running this script on _every_ `composer install`?

Reply:
>  Are there any strings in action-scheduler that need replaced?

Yes for PHP, probably no for js.

Reply:
> I think it might be good to comment out at least one of these lines instead of removing it. That way this script still has the prior art that can be used to inform anything in the future

Done in https://github.com/woocommerce/woocommerce/pull/42017/commits/e0c15ac930e5102a5a8109cb805e58eb2100a760

Reply:
@ObliviousHarmony This is ready for a second review.